trace-sdk/src/test/java/io/bitrise/trace/data/management/formatter/application/ApplicationVersionCodeDataFormatterTest.java

```diff
@@ -0,0 +1,32 @@
+package io.bitrise.trace.data.management.formatter.application;
+
+import org.junit.Test;
+
+import io.bitrise.trace.data.collector.application.ApplicationVersionCodeDataCollector;
+import io.bitrise.trace.data.dto.Data;
+import io.bitrise.trace.data.dto.FormattedData;
+import io.bitrise.trace.data.management.formatter.BaseDataFormatterTest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Unit tests for {@link ApplicationVersionCodeDataFormatter}.
+ */
+public class ApplicationVersionCodeDataFormatterTest extends BaseDataFormatterTest {
```

Comment:
Should we add a case for null content?

Reply:
We already have this scenario covered in the [ResourceDataFormatterTest](https://github.com/bitrise-io/trace-android-sdk/blob/main/trace-sdk/src/test/java/io/bitrise/trace/data/management/formatter/ResourceDataFormatterTest.java#L75) - as all resources *could* be null, we added that test much earlier. Does that make sense?

Reply:
Looks good, thank you!